fusor/model.py

```diff
@@ -30,6 +32,51 @@ def _is_curie(value):
     return v
 
 
+class DomainStatus(str, Enum):
+    """Define possible statuses of critical domains."""
+
+    LOST = "lost"
+    PRESERVED = "preserved"
+
+
+class CriticalDomain(BaseModel):
+    """Define CriticalDomain class"""
+
+    status: DomainStatus
+    name: str
+    id: str
```

Comment:
This is just a preference thing, but can we get in the habit of using StrictStr?